src/main/java/org/zalando/nakadi/service/subscription/SubscriptionService.java

```diff
@@ -197,9 +199,11 @@ public Result listSubscriptions(final String owningApplication, final Set<String
 
     private List<SubscriptionEventTypeStats> createSubscriptionStat(final Subscription subscription)
             throws ServiceUnavailableException {
+        final String subscriptionId = subscription.getId();
         final ZkSubscriptionClient zkSubscriptionClient =
-                zkSubscriptionClientFactory.createZkSubscriptionClient(subscription.getId());
-        final Partition[] partitions = zkSubscriptionClient.listPartitions();
+                zkSubscriptionClientFactory.createZkSubscriptionClient(subscriptionId);
+        final ZkSubscriptionNode zkSubscriptionNode = zkSubscriptionClient.getZkSubscriptionNode();
```

Comment:
@v1ctor I am locking it inside this method call https://github.com/zalando/nakadi/pull/427/files/ec2e3f7ad4170463e46576fa2abb1386c3bf6f86#diff-b07971694c861bf3a96d82f76f5459bdR299